packages/expo-localization/src/ExpoLocalization.web.ts

```diff
@@ -1,46 +1,47 @@
-// @flow
 import * as rtlDetect from 'rtl-detect';
+import { Localization } from './Localization.types';
+
+declare const navigator: Navigator;
```

Comment:
The declarations shouldn't be necessary - the TS config is set to enable web definitions already, which define window and navigator globally. I tested `window....` in a test module (expo-asset) and VS Code + yarn build were fine without an extra declaration.